src/scss/skin-modern/components/_seekbar.scss

```diff
@@ -83,13 +83,20 @@ $seekbar-height: .3125em;
       > .#{$prefix}-seekbar-marker {
         @extend %bar;
 
-        border-right: $marker-width solid $color-primary;
-        // don't consider the border for width, else it does not overlap at edges
-        box-sizing: content-box;
+        background-color: $color-primary;
         height: 100%;
-        // offset position marker to center it on its actual position
-        margin-left: -$marker-width / 2;
-        width: 0;
+        width: $marker-width;
+        text-align: center;
+        transition-property: all;
+        transition-duration: 1s;
```

Comment:
Nit: by adding `transition-timing-function: linear;` you should be able to have the transition 100% in sync with the seek bar :smiley:

Reply:
Addressed in https://github.com/bitmovin/bitmovin-player-ui/pull/318/commits/7df9bbbb158c4f28452ca5d4d8ae6e1cd6466744